config/locales/find.yml

```diff
@@ -113,8 +113,7 @@ en:
             <p class="govuk-body">To do this, you’ll need to be sponsored by your training provider.</p>
             <p class="govuk-body">Before you apply for this course, contact us to check Student visa sponsorship is available. If it is, and you get a place on this course, we’ll help you apply for your visa.</p>
       entitlement:
-        html:
-          <p class="govuk-body">You may be entitled to £10,000 from the UK government to help with the financial costs of moving to England.</p>
+        html: <p class="govuk-body">You may be entitled to £10,000 from the UK government to help with the financial costs of moving to England.</p>
```

Comment:
Did you change this by mistake?

Reply:
I think it got picked up by prettier. Should they not be on the same line anyway?

Reply:
disabled prettier and changed it back 👍